node-runner-cli/api/ValidatorConfig.py

```diff
@@ -108,12 +108,30 @@ def add_change_ownerid(actions: List, validator_info: EntityResponse):
         owner_id = input("\nEnter the new owner id or press Enter not to change:").strip()
         if owner_id != "":
             if owner_id != current_value:
-                actions.append(Action().set_validator_owner(owner_id))
+                actions.append(Action.set_validator_owner(owner_id))
                 return actions
             Helpers.print_coloured_line("Owner entered is same . So action will not be applied", bcolors.WARNING)
 
         return actions
 
+    @staticmethod
+    def vote(actions: List, validator_info: EntityResponse, health, engine_configuration):
+        print("-------- Vote -----\n")
```

Comment:
The neat part of my head is annoyed that the number of dashes on the left and right of this are uneven :')